index.html

```diff
@@ -220,6 +222,42 @@ <h3>Multi Post Stories Gain+Glory</h3>
     </div>
   </section>
 
+  <section class="modal-section" id="modal">
+    <div class="modal-container">
+      <div class="modal-content">
+        <div class="modal-image">
+          <div class="times" id="close-btn" onclick>&times;</div>
+          <img id="prj-img" src="images/mobile-popup.png" alt="mobile-popup" class="img" />
+          <img id="prj-img-1" src="images/destkop-popup.png" alt="destkop-popup" class="img" />
+        </div>
+        <div class="modal-title">
+          <h3 class="about-heading modal-heading">Keeping track of hundreds of
+            components</h3>
+        </div>
+        <ul class="modal-list">
+          <li>Ruby</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>HTML</li>
+        </ul>
+        <div class="modal-paragraph">
+          <p class="modal-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown
+            printer took a galley of type and scrambled it 1960s.</p>
+        </div>
+        <div class="btn">
+          <a href="" class="f-icon-1">See Live <img src="images/external-icon.png" alt="external-icon"></a>
+          <a href="" class="f-icon-2">See Source<img src="images/githubvector.png" alt="github-icon"></a>
```

Comment:
- [ ]  Your project must be responsive but unfortunately, it's not responsive. Please have another look at the project requirements, 
<img width="306" alt="Capture" src="https://user-images.githubusercontent.com/58504655/128069429-dbce41db-adf6-4384-a167-a72781bbde7d.PNG">